app/log.py

```diff
@@ -62,4 +63,8 @@ def setup(config: Config) -> None:
             dsn=config.sentry_dsn.get_secret_value(),
             traces_sample_rate=1.0,
             profiles_sample_rate=1.0,
+            profile_lifecycle="trace",
+            integrations=[
+                AsyncioIntegration(),
+            ],
```

Comment:
```suggestion
            integrations=[AsyncioIntegration()],
```
Feel free to disagree of course.